extensions/panache/hibernate-orm-panache/runtime/src/main/java/io/quarkus/hibernate/orm/panache/PanacheEntityBase.java

```diff
@@ -257,6 +258,18 @@ public void flush() {
         throw JpaOperations.implementationInjectionMissing();
     }
 
+    /**
+     * Find all entities of this type, in the given criteria.
+     *
+     * @param criteria to use
+     * @return a new {@link PanacheQuery} instance to find all entities of this type.
+     * @see #findAll()
+     */
+    @GenerateBridge
+    public static <T extends PanacheEntityBase> PanacheQuery<T> findAll(CriteriaQuery<T> criteria) {
```

Comment:
This should better be in `find` because `CriteriaQuery` can have `WHERE` clauses and so it's not necessarily "all results". Furthermore, it would likely need the equivalent `list` and `stream` shortcuts.